manifests/s/Smile/TextExpander/7.2/Smile.TextExpander.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Smile.TextExpander
+PackageVersion: "7.2"
```

```suggestion
PackageVersion: "254.7.2.16"
```
This must match Control Panel.

You'll also need to update the version number in the package version folder to reflect this.